doc/manual/installing.rst

```diff
@@ -73,12 +83,17 @@ Installing multiple packages and making them visible to the ARTIQ commands requi
           ];
         };
       };
+
+    nixConfig = {
+      extra-trusted-public-keys = "nixbld.m-labs.hk-1:5aSRVA5b320xbNvu30tqxVPXpld73bhtOeH6uAjRyHc=";
+      extra-substituters = "https://nixbld.m-labs.hk";
+    };
```

Comment:
Doesn't look like it. I just did a quick check in a clean docker image (nixos/nix:2.11.0) following the current ARTIQ manual and it's building everything locally (https://pastebin.com/bzzEMNab). Doesn't ask about allowing the settings either.

Reply:
Yep, I've also run into this; the settings don't appear to be transitive. (Nix is really under-documented in this area, though!)

Reply:
A nice thing to have, but it wouldn't fix this issue since the ARTIQ flake is a child flake in this scenario. As the instructions currently stand, M-Labs' binary cache for the "stable" release 7 builds is essentially useless.

Reply:
A nice thing to have, but until (if) it gets implemented, M-Labs' binary cache is essentially useless if users follow the docs.